src/renderer/features/currency/ChangeCurrency/model/currency-form.ts

```diff
@@ -0,0 +1,63 @@
+import { createEvent, createStore, sample, forward } from 'effector';
+
+import { currencyModel, priceProviderModel } from '@renderer/entities/price';
+import { CurrencyItem } from '@renderer/shared/api/price-provider';
+
+export const $activeCurrency = createStore<CurrencyItem['id']>(0);
+export const $fiatFlag = createStore<boolean>(false);
+
+const currencyChanged = createEvent<CurrencyItem['id']>();
+const fiatFlagChanged = createEvent<boolean>();
+
+const submitForm = createEvent();
+const resetValues = createEvent();
+
+// initial values
+sample({
+  clock: [currencyModel.$activeCurrency, resetValues],
+  source: currencyModel.$activeCurrency,
+  fn: (currency) => currency?.id || 0,
+  target: $activeCurrency,
+});
+
+sample({
+  clock: [priceProviderModel.$fiatFlag, resetValues],
+  source: priceProviderModel.$fiatFlag,
+  fn: Boolean,
+  target: $fiatFlag,
+});
+
+// on change
+forward({
+  from: currencyChanged,
+  to: $activeCurrency,
+});
+
+forward({
+  from: fiatFlagChanged,
+  to: $fiatFlag,
+});
+
+// save changes
+sample({
```

Comment:
Because we don't have a unified form we have to call `changed` events one by one, that's fine
But there is a much cleaner way with [effector/patronum](https://patronum.effector.dev/methods/spread/#call-events-on-store-changes)
It would look something like this:
```javascript
sample({
  clock: formSubmitted,
  source: { fiatFlag: $fiatFlag, activeCurrency: $activeCurrency },
  target: spread({
    targets: {
      fiatFlag: priceProviderModel.events.fiatFlagChanged,
      activeCurrency: currencyModel.events.currencyChanged,
    }
  })
})
```
I've never used it before, so If it doesn't work, please let me know 😉

Reply:
Overall I assume it would be right to use `effector-forms` even though we have 2 fields without any inputs.
With form it would be easier to add new fields (store + event + validation) instead of doing it by hand
I was wrong guessing that 2 non-input fields are ok to not introduce the form